.backportrc.json

```diff
@@ -2,7 +2,7 @@
     "repoOwner": "mongodb",
     "repoName": "docs-cluster-to-cluster-sync",
   
-    "targetBranchChoices": ["master", "v1.9", "v1.8"],
+    "targetBranchChoices": ["master", "v1.10", "v1.9", "v1.8"],
```

Comment:
I might be missing something here, but I don't see branch "v1.10" in the list of [c2c repo branches](https://github.com/mongodb/docs-cluster-to-cluster-sync/branches/all). It looks like we recently created branch [v.10](https://github.com/mongodb/docs-cluster-to-cluster-sync/tree/v.10). Was this maybe a typo and should be v1.10 instead?

Reply:
Ohno looks like that was a typo on my end. Just fixed it on Github and redeployed. It should be `v1.10` now

Reply:
Nice!